runtime/syscall-server/syscall_context.cpp

```diff
@@ -46,6 +50,37 @@
 			    .imm = IMM })
 #endif
 
+#if linux && !BPFTIME_BUILD_WITH_LIBBPF
```

Comment:
Is `linux` a correct macro...?

Reply:
I might have messed it up here, will correct it. Thanks for pointing it out.